legend-engine-pure-code-compiled-core/src/main/resources/core/store/m2m/tests/legend/simpleObject.pure

```diff
@@ -801,6 +801,27 @@ meta::pure::mapping::modelToModel::test::alloy::simple::testMappingWithTypeName(
    assert(jsonEquivalent('{"firstName":"Pierre","lastName":"Doe","type":"_S_Person","description":"meta::pure::mapping::modelToModel::test::shared::src::_S_Person"}'->parseJSON(), $result.values->toOne()->parseJSON()));
 }
 
+function <<meta::pure::profiles::test.Test, meta::pure::profiles::test.AlloyOnly>> 
```

Comment:
@aziemchawdhary-gs can we have a test with test data so that filter is executed or maybe just source tree calculation test which demonstrates that property is not added for both trees and only for one having filter?
Or do you think execution of such mapping is good enough?

Reply:
good suggestion - I have added a test for source tree calculation